### ISSUE TITLE
Develop Siamese Loss Function For Generation from Clusters

### ISSUE THREAD

**Issue by garnav** [CornellDataScience/IntSys-Sentiment-Summary#16]
Check if the following is possible:
Assuming that we're given an autoencoder, use (a) the embeddings it produces and (b) clustered sentences to train a siamese neural network.
